phpmyadmin/stack.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.1'
+
+services:
+  db:
+    image: mariadb:10.3
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: notSecureChangeMe
+
+  phpmyadmin:
+    image: phpmyadmin
```

Comment:
It seems that perhaps `:latest` being missing from https://github.com/docker-library/official-images/pull/7288 was the oversight? :sweat_smile:

(Did you want to include `restart: always` on this container as well so that it matches the `db` container?)